### ISSUE TITLE
https://arxiv.org/abs/1809.06065

### ISSUE THREAD

**Issue by Swall0w** [Swall0w/papers#673]
Peng Yun, Lei Tai, Yuan Wang, Ming Liu

3D object detection is still an open problem in autonomous driving scenes. Robots recognize and localize key objects from sparse inputs, and suffer from a larger continuous searching space as well as serious fore-background imbalance compared to the image-based detection. In this paper, we try to solve the fore-background imbalance in the 3D object detection task. Inspired by the recent improvement of focal loss on image-based detection which is seen as a hard-mining improvement of binary cross entropy, we extend it to point-cloud-based object detection and conduct experiments to show its performance based on two different type of 3D detectors: 3D-FCN and VoxelNet. The results show up to 11.2 AP gains from focal loss in a wide range of hyperparameters in 3D object detection. Our code is available at this https URL.

https://arxiv.org/abs/1809.06065
